integration-tests/src/test/java/org/zowe/apiml/integration/proxy/UiIntegrationTest.java

```diff
@@ -19,13 +19,13 @@
 import org.zowe.apiml.util.http.HttpRequestUtils;
 
 import static io.restassured.RestAssured.given;
+import static org.zowe.apiml.util.requests.Endpoints.*;
 
 @DiscoverableClientDependentTest
 class UiIntegrationTest implements TestWithStartedInstances {
     protected static String[] discoverableClientSource() {
         return new String[]{
-            "/discoverableclient/ui/v1",
-            "/ui/v1/discoverableclient"
+            STATIC_UI
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed